### PR TITLE
I added POM page for EditAccountProfile and I added 1 element to POM …

### PR DESCRIPTION
**Pull request by JuliaEde**
…ViewProfilePage (the element takes the unordered list from hamburger menu, I am not sure if I should use get or wrap it in an async, because in order to get this element, the hamburger button has to be clicked)

About the new EditAccountProfile POM page: I added all elements from the Edit Account Page, the only problem that I have is this line - return super.open('/user/61a639262d347ec0559fe77d/edit'); I don't know how to get User Id, that's why I created the new element in the ViewProfile.page.js, because it has the link to User profile with the User ID

### CODE REVIEW

File: test/pageobjects/EditAccountProfileJuliaE.page.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const Page = require('./Page');
+class EditAccountProfileJuliaEPage extends Page {
```
Comment (ponomid) [inline test/pageobjects/EditAccountProfileJuliaE.page.js:2]: Should be empty line between first and second

Comment (JuliaEde) [reply test/pageobjects/EditAccountProfileJuliaE.page.js:2]: Илья, а можно прямо на сайте редактировать код?

Comment (JuliaEde) [reply test/pageobjects/EditAccountProfileJuliaE.page.js:2]: Или нужно через вебшторм?

Comment (ponomid) [reply test/pageobjects/EditAccountProfileJuliaE.page.js:2]: > Илья, а можно прямо на сайте редактировать код?

Можно и здесь. Переходишь в File changed > Edit File

Comment (JuliaEde) [reply test/pageobjects/EditAccountProfileJuliaE.page.js:3]: Кажется получилось отредактировать

File: test/pageobjects/EditAccountProfileJuliaE.page.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        return $('#languages');
+    }
+    get btnCancel() {
+        return $("//button[@class='MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root mt-5 mr-3 css-79xub']");
+    }
+    get btnSave() {
+        return $("//span[@class='MuiButton-startIcon MuiButton-iconSizeMedium css-6xugel']");
+    }
+
+    open() {
```
Comment (yuliahar) [inline test/pageobjects/EditAccountProfileJuliaE.page.js:29]: Please comment this method for now.

File: test/pageobjects/EditAccountProfileJuliaE.page.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const Page = require('./Page');
+
+class EditAccountProfileJuliaEPage extends Page {
```
Comment (yuliahar) [inline test/pageobjects/EditAccountProfileJuliaE.page.js:3]: Please rename this class to EditAccountProfilePage

File: test/pageobjects/EditAccountProfileJuliaE.page.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        return $('#image');
+    }
+    get inputAbout() {
+        return $('#about');
+    }
+    get inputProgrammingLanguages() {
+        return $('#languages');
+    }
+    get btnCancel() {
+        return $("//button[@class='MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root mt-5 mr-3 css-79xub']");
```
Comment (yuliahar) [inline test/pageobjects/EditAccountProfileJuliaE.page.js:23]: Please try to find a locator without so many CSS classes

File: test/pageobjects/EditAccountProfileJuliaE.page.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    get btnSave() {
+        return $("//span[@class='MuiButton-startIcon MuiButton-iconSizeMedium css-6xugel']");
```
Comment (yuliahar) [inline test/pageobjects/EditAccountProfileJuliaE.page.js:26]: Please try to find a locator with fewer amount CSS classes.

File: test/pageobjects/ViewProfile.page.js [hunk 6]
```diff
@@ -12,6 +12,9 @@ class ViewProfilePage extends Page {
     get btnProfile() {
         return $("#profile");
     }
+    get hamburgerUnorderedList() {
+        return $("//ul[@class='MuiList-root MuiList-padding css-1ontqvh']");
+    }
     async profilePage () {
```
Comment (yuliahar) [inline test/pageobjects/ViewProfile.page.js:18]: Please rename this method to goToProfilePage

Comment (yuliahar) [reply test/pageobjects/ViewProfile.page.js:18]: Add comment open method for now.

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: test/pageobjects/EditAccountProfileJuliaE.page.js (hunk 3 above)
Comment (JuliaEde) [reply test/pageobjects/EditAccountProfileJuliaE.page.js:3]: Done, renamed

File: test/pageobjects/EditAccountProfileJuliaE.page.js (hunk 2 above)
Comment (JuliaEde) [reply test/pageobjects/EditAccountProfileJuliaE.page.js:29]: Done, open method commented

File: test/pageobjects/EditAccountProfileJuliaE.page.js (hunk 4 above)
Comment (JuliaEde) [reply test/pageobjects/EditAccountProfileJuliaE.page.js]: Done, renamed

File: test/pageobjects/EditAccountProfileJuliaE.page.js (hunk 5 above)
Comment (JuliaEde) [reply test/pageobjects/EditAccountProfileJuliaE.page.js]: Done

**Pull request merged**